2-mandatory/1-create-functions.js

```diff
@@ -3,17 +3,30 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(array) {
+  const newArray = [];
+  const numberOfElements = Math.min(array.length, 5); 
+  for (let i = 0; i < numberOfElements; i++) {
+    newArray.push(array[i]);
+  }
+  return newArray;
 }
 
+//return array.slice(0, 5);
```

Comment:
your loop works great, but based on this comment it looks like you also found an easier way to do it :)